uint64 integral_dt	# integration time
uint64 error_count
float32 x		# angular velocity in the NED X board axis in rad/s
float32 y		# angular velocity in the NED Y board axis in rad/s
float32 z		# angular velocity in the NED Z board axis in rad/s
float32 x_integral	# delta angle in the NED X board axis in rad/s in the integration time frame
float32 y_integral	# delta angle in the NED Y board axis in rad/s in the integration time frame
float32 z_integral	# delta angle in the NED Z board axis in rad/s in the integration time frame
float32 temperature	# temperature in degrees celcius
float32 range_rad_s
float32 scaling

int16 x_raw
int16 y_raw
int16 z_raw
int16 temperature_raw

uint32 device_id	# unique device ID for the sensor that does not change between power cycles
